src/main/java/seedu/dengue/model/person/Date.java

```diff
@@ -16,18 +19,28 @@
 public class Date {
 
 
-    public static final String MESSAGE_CONSTRAINTS = "Dates should be of the format yyyy-MM-dd"
-            + "and adhere to the following constraints:\n"
-            + "1. Date to be formatted as yyyy-MM-dd in 3 blocks separated by a hyphen '-'\n"
-            + " - Each block should only contain digits, and no other special characters\n"
+    public static final String MESSAGE_CONSTRAINTS = "Dates should adhere to the following constraints:\n"
+            + "1. Dates to contain 4-digit years YYYY, 2-digit days and either 2-digit months, 3-letter months, \n"
+            + "or fully-spelled months.\n"
+            + " - If months are given in the MM format and the dates are ambiguous, eg. 20001010 \n "
+            + "YYYY will be read first, followed by MM and DD. \n"
+            + " - If months are given in MMM or MMMM formats, months will be read first.\n"
+            + " - eg. march 20 10 february will be read as march \n"
             + "2. Dates must be valid. i.e.\n"
             + " - MM must be a number between 01 and 12 inclusive\n"
```

Comment:
This constraint should be modified or removed.

Reply:
done